src/dune_rules/dep_conf.ml

```diff
@@ -51,11 +48,12 @@ let decode =
       ; ("alias", sw >>| fun x -> Alias x)
       ; ("alias_rec", sw >>| fun x -> Alias_rec x)
       ; ( "glob_files"
-        , sw >>| fun x -> Glob_files { glob = x; recursive = false } )
+        , sw >>| fun glob -> Glob_files Glob_files.{ glob; recursive = false }
```

Comment:
Should be `{ Glob_files.glob ; recursive = false }`

Reply:
Is including the module name explicitly here just to make the code more clear and so it doesn't look like the records fields are arguments to the `Glob_files` constructor?

Do we use the convention `{ Module_name.first_field; ... }` over `Module_name.{ first_field; ... }`?

Reply:
> Do we use the convention { Module_name.first_field; ... } over Module_name.{ first_field; ... }?

They have slightly different meanings. The former is for qualifying labels when constructing/matching records, while the latter is a local open - equivalent to `let open Module_name in { first_field ; ... }`

Reply:
Right that makes sense. Is the former preferred because using a local open would do more than is strictly necessary, and it might take longer to compile or bring unwanted things into scope?